lib/ex_doc/formatter/epub/templates/content_template.eex

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf"
+         unique-identifier="project-<%= config.project %>"
+         version="3.0">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:title><%= config.project %> - <%= config.version %></dc:title>
+    <dc:identifier id="project-<%= config.project %>"><%= uuid %></dc:identifier>
+    <dc:language>en-US</dc:language>
+    <meta property="dcterms:modified"><%= datetime %></meta>
+  </metadata>
+  <manifest>
+    <item id="ncx" href="toc.ncx" media-type="application/x-dtbncx+xml"/>
+    <item id="nav" href="nav.xhtml" media-type="application/xhtml+xml" properties="nav scripted"/>
+    <item id="cover" href="title.xhtml" media-type="application/xhtml+xml" properties="scripted"/>
+    <%= for extra <- config.extras do %>
+      <item id="<%= extra_title(extra) %>" href="<%= extra_title(extra) %>.xhtml" media-type="application/xhtml+xml" properties="scripted"/>
+    <% end %>
+    <%= for node <- nodes do %>
+      <item id="<%= node.id %>" href="<%= node.id %>.xhtml" media-type="application/xhtml+xml" properties="scripted"/>
+    <% end %>
+    <% alias ExDoc.Formatter.HTML.Templates, as: T %>
```

Comment:
Better to move the alias to the module that uses the templates, that way we avoid a blank line at the beginning of this XML file (`epubchecker` emits a warning for this).